### PR TITLE
chore(frontend): upgrade wagmi to 1.3.10

### PR DESCRIPTION
**Pull request by boriskubrik**
### Changes Made
 - Upgrades wagmi to version 1.3, installs it's new peer dependency viem.
 - Replaces all instances of ethers utils with viem equivalents
 - Removes ethers dependency

### Testing

 - [X] Approve button shows when token approval is less than send amount
 - [X] Approving tokens
 - [X] Approval button disappears after successful token approval
 - [x] Shifting from tokens
 - [x] Shifting from native tokens
 - [X] Injected wallet connection
 - [X] Coinbase wallet connection
 - ~~WalletConnect wallet connection~~ (needs fixing separately)

### CODE REVIEW

**Comment by immutablegoblin** [comment]
Build is broken as we need Typescript 5 I believe

**Review by bill0x2A — APPROVED** [review]

**Review by boriskubrik — COMMENTED** [review]
Code looks good, works during local testing. I can't approve as the author, need to get approval from @xykota.

**Review by immutablegoblin — APPROVED** [review]

**Pull request merged**
